.eslintrc.js

```diff
@@ -3,6 +3,7 @@ module.exports = {
   ignorePatterns: ['!.storybook'],
   rules: {
     'react/prop-types': 'off',
+    '@typescript-eslint/no-non-null-assertion': 'off',
```

Comment:
Should we disable only in the component but not the whole codebases?

Reply:
@hanam1ni I've updated it here. I put the check and then disable this rule. I don't remember why I added it in the first place. I'll check later in the next PR - [1ec439e](https://github.com/rosle/react-nimble-survey/pull/52/commits/1ec439e5da2ff238e1ad01a5d5921e6087c4cd99) 👍